src/request.rs

```diff
@@ -27,7 +27,7 @@ pub fn send_request(signed_request: &SignedRequest) -> Response {
         hyper_headers.set_raw(h.0.to_owned(), h.1.to_owned());
     }
 
-    let mut final_uri = format!("https://{}{}", signed_request.hostname(), signed_request.canonical_uri());
+    let mut final_uri = format!("http://{}{}", signed_request.hostname(), signed_request.canonical_uri());
```

Comment:
Oops! I must have been fiddling with that locally and forgot to revert that change.